gift-certificates-app/dao/src/main/resources/properties/application.properties

```diff
@@ -5,3 +5,4 @@ spring.jpa.properties.hibernate.current_session_context_class=org.springframewor
 
 
 
+
```

Comment:
Let's keep only one line in the end of file

Reply:
done